src/services/ldap/lockingQueue.unit.test.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable promise/no-callback-in-promise */
+const chai = require('chai');
+const LockingQueue = require('./lockingQueue');
+
+const { expect } = chai;
+
+describe('lockingQueue', () => {
+	it('should lock and unlock', (done) => {
+		const lockingQueue = new LockingQueue();
+		const promise1 = lockingQueue.getLock();
+		expect(lockingQueue.locked).to.be.true;
+		promise1
+			.then(() => {
+				expect(lockingQueue.locked).to.be.false;
+				return done();
+			})
+			.catch(() => {});
+		lockingQueue.releaseLock();
+		expect(lockingQueue.locked).to.be.false;
```

Comment:
when done() from above is executed (by releaseLock()), this line must not be covered by the test. suggestion: one test for lock, another one for unlock.